### PR TITLE
Added Check To See If BASH Exist's In The CHROOT

### PR DESCRIPTION
**Pull request by jcjordyn130**
This New Code Is A Check To See If BASH Exist's In The Chroot, If It Does Run Bash By Default, Else It Fall's Back To /bin/sh

### CODE REVIEW

**Comment by kyrias** [comment]
That is not how you spell in English, or all other languages I know of.

File: arch-chroot.in [hunk 1]
```diff
@@ -52,4 +52,9 @@ shift
 chroot_setup "$chrootdir" || die "failed to setup chroot %s" "$chrootdir"
 chroot_add_resolv_conf "$chrootdir" || die "failed to setup resolv.conf"
 
+if [ -e $chrootdir/bin/bash ]
```
Comment (kyrias) [inline arch-chroot.in]: Don't use `test` in bash scripts

File: arch-chroot.in [hunk 2]
```diff
@@ -52,4 +52,9 @@ shift
...
+then
```
Comment (kyrias) [inline arch-chroot.in]: Don't place then on it's own line

File: arch-chroot.in [hunk 3]
```diff
@@ -52,4 +52,9 @@ shift
...
+SHELL=/bin/bash unshare --fork --pid chroot "$chrootdir" "$@"
```
Comment (kyrias) [inline arch-chroot.in]: Indent if clauses.

File: arch-chroot.in [hunk 4]
```diff
@@ -52,4 +52,9 @@ shift
...
+else
 SHELL=/bin/sh unshare --fork --pid chroot "$chrootdir" "$@"
```
Comment (kyrias) [inline arch-chroot.in]: Indent else clauses.

**Comment by falconindy** [comment]
This change is a mess... I'd sooner just switch the default to /bin/bash, since bash *will* be on any sane Arch install. So, I've done that in 69081c9e7ee90.

**Comment by jcjordyn130** [comment]
good to see i gave to the community

**Pull request closed without merging**